packages/generic-utils/src/create-deferred/index.ts

```diff
@@ -0,0 +1 @@
+export * from './create-deferred'
```

Comment:
Hmm I thought having a directory for every utility would make sense, since while more verbose, it allows more flexibility when it comes to grouping than a list of flat files

Reply:
I like to keep them separate. Something very confusing in AJS' codebase is that a lot of things are only in `index` files so all my tabs look the same

Reply:
I concur about ajs -- my favorite is when there is an index.ts that both re-exports files _and_ itself contains code 🤪

Reply:
Yeah, maybe it'll be more clear to me once we have more than a single utility - right now I'm wondering either why we need the 2nd-level `index.ts` at all, or even the directory (could have `src/create-deferred.ts` instead), but ultimately it doesn't matter too much since people will be using bundlers where file reads add up anyway. Happy to yield on this one and mark it resolved!